docs/update_and_migration/from_4.6/update_from_4.6.md

```diff
@@ -391,16 +395,16 @@ To use the [latest features](ibexa_dxp_v4.6.md) added to them, update them separ
 
     Then apply manually the changes described below.
 
-    ## 4.6.20
+    ### Discounts 4.6.20
```

Comment:
It disturb me to not have the 'v' on those ones contrary to the rest of the ToC above.
```suggestion
    ### Discounts v4.6.20
```